project/scripts/fetch_trends.py

```diff
@@ -92,6 +95,32 @@ def aggregate_hourly_to_daily(hourly_df):
     return new_data
 
 
+def verify_data_complete(daily_data, entity):
```

Comment:
May be the name can be clarified. Verify makes it seem that a boolean is returned

Reply:
does ensure sound better?

Reply:
backfill_missing_data_as_necessary?

Reply:
sure